src/main/java/com/google/cloud/spanner/pgadapter/parsers/ArrayParser.java

```diff
@@ -212,8 +214,6 @@ private static Object toValidSpannerElement(@Nonnull Object value, int elementOi
         return ((Short) value).longValue();
       case Oid.INT4:
         return ((Integer) value).longValue();
-      case Oid.FLOAT4:
```

Comment:
This is being removed now, as these are data types that needed special handling because:
1. We allow PostgreSQL clients to send us `int2[]`, `int4[]` and `float4[]` arrays.
2. These are converted to `int8[]` and `float8[]` before being sent to Cloud Spanner.
3. This conversion is no longer needed for `float4` when Cloud Spanner supports `float4` natively.